packages/components/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -148,6 +157,7 @@ export const ConfirmationModal = forwardRef(function ConfirmationModalInternal(
     onConfirm,
     onCancel,
     onRequestClose,
+    actionType,
```

Comment:
can we set a default here? We _usually_ want a default value for most optional props.

```tsx
...
onRequestClose
actionType = 'work'
...
```

Reply:
I think I'd default to `work`, if only to avoid this being a breaking change